devel/libhv/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+PortGroup           openssl 1.0
+
+github.setup        ithewei libhv 1.3.1 v
+github.tarball_from archive
+revision            0
+
+categories          devel
+maintainers         {@sikmir disroot.org:sikmir} openmaintainer
+
+license             BSD
+
+description         A c/c++ network library for developing TCP/UDP/SSL/HTTP/WebSocket/MQTT client/server
+long_description    {*}${description}
+
+checksums           rmd160  27c3bb6613ede4ae8ce5c685b7987a41eeed70b0 \
+                    sha256  da4757c3520854cf2081f796733f7bc023651d60baab448a06fcf66435a859be \
+                    size    880367
+
+installs_libs       no
```

Comment:
Also, this is incorrect.  This port absolutely installs libraries - that's its very purpose 😆 

Library installed: `/opt/local/lib/libhv.dylib`

Reply:
Sure, that's copy-past. Removed.